src/app/components/normal-component/demo-page/demo-page.component.spec.ts

```diff
@@ -16,7 +17,21 @@ describe('DemoPageComponent', () => {
     });
 
     it('should display demo nodes', () => {
-        expect(testUtils.findElements('app-demo-card').length).withContext('app-demo-card should be present').toBeGreaterThan(0);
+        const numberOfDemos: number = testUtils.findElements('app-demo-card').length;
+        expect(numberOfDemos)
+            .withContext('app-demo-card should be present')
+            .toBeGreaterThan(0);
+    });
+
+    it('should have demo nodes with step being a move', () => { // Yep, this happend
```

Comment:
"this happend" -> "this happened", mais ce commentaire est pas très utile, peut-être expliquer en commentaire quel était le bug plutôt que de juste dire "c'est arrivé"
Ici je comprends pas le it, un demo node avec une étape qui est un move, waaat?

Reply:
le test correspond pas au it btw, on teste qu'il n'y a pas de nœud qui est son propre parent